walkbox.go

```diff
@@ -57,14 +57,14 @@ func (w *WalkBox) isConvex() bool {
 	return totalCrossProduct != 0
 }
 
-// Enable sets the enabled state of the WalkBox.
-func (w *WalkBox) Enable(enable bool) *WalkBox {
-	w.enabled = enable
-	return w
-}
-
 // ContainsPoint check if the provided position is in the boundaries defined by the WalkBox.
 func (w *WalkBox) ContainsPoint(p *Positionf) bool {
+	// Check if the position is one of the vertices
```

Comment:
We need this cause, same edge not intersect with itself but, it is still being part of the polygon